app/Http/Controllers/BankIntegrationController.php

```diff
@@ -521,23 +524,45 @@ public function bulk(BulkBankIntegrationRequest $request)
      */
     public function refreshAccounts(AdminBankIntegrationRequest $request)
     {
-        // As yodlee is the first integration we don't need to perform switches yet, however
-        // if we add additional providers we can reuse this class
+        $account = auth()->user()->account;
+
+        $this->refreshAccountsYodlee($account);
```

Comment:
due to the return response in this method, code beneath it may never run, i'd suggest just returning if no account_id is present